packages/common/nbstore/src/worker/ops.ts

```
@@ -140,6 +140,10 @@ interface GroupedWorkerOps {
       AggregateResult<any, any>,
     ];
   };
+  sync: {
+    enableBatterySaveMode: [void, void];
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)